BUILD_MESON

```diff
@@ -122,20 +129,37 @@ Enable "unit-tests" option to run the test suite:
      
 Unlike Dinit's Make build system, We don't enable -fsanitize by default so if you need that use this option in meson setup step:
 
-     meson setup -Db_sanitizes='address,undefined' dirbuild/
+     meson setup -Db_sanitizes='address,undefined' builddir/
 
 Enable "igr-tests" to run the integration tests:
 
-     meson setup -Digr-tests=true dirbuild/
+     meson setup -Digr-tests=true builddir/
 
 (The integration tests are more fragile than the unit tests, but give a better indication that
 Dinit will actually work correctly on your system).
 
-Finally switch current directory to Your builddir (eg dirbuild), compile Dinit with meson compile and run tests via:
+Finally compile Dinit with "meson compile -C builddir" and run tests via:
+
+     meson test -C builddir
 
-     meson test
+Then Meson report test status. All test logs available at "builddir/meson-logs/testlog.txt".
 
-Then Meson report test status.
+
+Installation
+=-=-=-=-=-=-
+
+You can install dinit via this command:
+
+     meson install -C builddir
```

Comment:
Does this work for "older versions" of meson?

Reply:
[Meson docs](https://mesonbuild.com/Installing.html) said:
> Installing
Invoked via the [following command](https://mesonbuild.com/Commands.html#install) (available since 0.47.0):
> ```
> meson install
> ```